### PR TITLE
Update ko.json

### PR DESCRIPTION
**Pull request by Aronako**
Edit Untranslated Lines

### CODE REVIEW

**Comment by Flashfyre** [comment]
Conflicts

**Comment by Aronako** [comment]
Fixed

File: lang/ko.json [hunk 1]
```diff
@@ -397,84 +397,84 @@
...
-            "name": "28",
-            "description": "２８"
+            "name": "28 뱃지",
+            "description": "선물상자의 내용물을 확인하기"
           },
           "blue_orb": {
-            "name": "Blue Orb",
-            "description": "Visit Scrambled Egg Zone"
+            "name": "파란 구슬 뱃지",
+            "description": "Blue Orb World 방문하기"
```
Comment (Flashfyre) [inline lang/ko.json:469]:
```suggestion
            "description": "Scrambled Egg Zone 방문하기"
```

**Review by Flashfyre — CHANGES_REQUESTED** [review]

File: lang/ko.json [hunk 2]
```diff
@@ -397,84 +397,84 @@
       }
     },
     "badges": {
-      "null": "No Badge",
+      "null": "뱃지 없음",
       "locked": "???",
       "location": "Location:&nbsp;",
```
Comment (Flashfyre) [inline lang/ko.json:402]: This should probably be translated too

**Review by Flashfyre — CHANGES_REQUESTED** [review]

**Review by Flashfyre — APPROVED** [review]

**Pull request merged**